target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -17,6 +17,9 @@ led_wlan="$(get_dt_led wlan)"
 [ -n "$led_wlan" ] && ucidef_set_led_wlan "wifi_led" "wifi" "$led_wlan" "phy0tpt"
 
 case $board in
+beeline,smartbox-turbo-plus)
+	ucidef_set_led_netdev "wan" "wan" "$boardname:green:status" "wan"
```

Comment:
Is this vendor behavior? If you use it for WAN, you should just drop the led-running in DTS, as that's useless/misleading then.

Reply:
We have been fighting this problem for a year now.
The source router displays WAN port activity in green.
No other internet connection is provided by the manufacturer.

Reply:
Do you think you need to delete?

Reply:
Well, either delete this line and use the LED for "successfully booted" indication, or remove the led-running line in aliases (DTS) and use the LED for WAN status indication.

Reply:
I choose to remove this line.
Maybe someone after me will offer a more elegant solution